did/utils.py

```diff
@@ -9,7 +9,6 @@
 import re
 import sys
 import unicodedata
-from pprint import pformat as pretty
```

Comment:
Oh, this can have a bad consequence as some library or file down the road will be missing `pretty`. I've remove this line once and moved it back in because I was too lazy to fix it properly.

Reply:
Good point! Actually, `pretty` is used by other modules. Fixed in 82dd820. Thanks!

Reply:
You're very welcome!